Source/MetroSystem/src/metroSystem/MetroSystem.java

```diff
@@ -29,7 +29,98 @@ public Language getSystemLanguage() {
         return systemLanguage;
     }
 
-    public Database getDatabase() {
-        return database;
+    public Database getDatabase() { return database; }
+
+    /**
+     * Get all edges in a line.
+     * @return  All edges in a line
```

Comment:
It is suggested to write the whole JavaDoc, instead of writing part of it...  
For example, adding `@param` for each parameter seems to be a better practice.